Documentation/installing-dependencies.rst

```diff
@@ -1,216 +1,293 @@
+Manual Installation
 -------------------
 
-Initial Adjustments to .bashrc
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend setting up a folder on your system which will contain the build files for all dependencies.
+For example, do `mkdir ~/seissol; cd seissol`.
+
+In _all_ cases before building something manually,
+make sure to check the installed module files. That is, type `module avail` and look for the software you want to use.
+Type `module load NAME` to load the respective software (with `NAME` being the name of the software, including the text after the slash, e.g. `module load cmake/3.20.0`).
 
-Add the following lines to your .bashrc (vi ~/.bashrc).
+Setting Helpful Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a file `setup.sh` with the following enviroment variables. The following script assumes that you use the folder `~/seissol`
+for building.
 
 .. code-block:: bash
 
   # For intel compiler
   # source /opt/intel/compiler/VERSION/bin/compilervars.sh intel64
   
-  export PATH=$HOME/bin:$PATH
-  export LIBRARY_PATH=$HOME/lib:$LIBRARY_PATH
-  export LD_LIBRARY_PATH=$HOME/lib:$LD_LIBRARY_PATH
-  export PKG_CONFIG_PATH=$HOME/lib/pkgconfig:$PKG_CONFIG_PATH
-  export CMAKE_PREFIX_PATH=$HOME
-  export EDITOR=vi
-  export CPATH=$HOME/include:$CPATH 
+  # write the path here which you created your directory in (you can figure it out by typing `pwd`)
+  export SEISSOL_BASE=$HOME/seissol
+
+  export PREFIX=$SEISSOL_BASE/local
+  export PATH=$PREFIX/bin:$PATH
+  export LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig:$PREFIX/lib64/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$PREFIX:$CMAKE_PREFIX_PATH
+  export CMAKE_INSTALL_PREFIX=$PREFIX
+  export CPATH=$PREFIX/include:$CPATH 
+  export C_INCLUDE_PATH=$PREFIX/include:$C_INCLUDE_PATH
+  export CXX_INCLUDE_PATH=$PREFIX/include:$CXX_INCLUDE_PATH
+  export EDITOR=nano # or e.g. vi,vim
+
+  # run "source ~/seissol/setup.sh" to apply environment to the current shell
 
-  # run "exec bash" or "source ~/.bashrc" to apply environment to the current shell
+Required Dependencies
+~~~~~~~~~~~~~~~~~~~~~
 
-Installing CMake
-~~~~~~~~~~~~~~~~
+We assume that you have a compiler already installed.
+The same goes for a suitable Python installation.
+You will also need CMake in version 3.20.0 or above. Most likely, you system will already have a
+version of CMake installed; however, you may have to load a module to get a new enough version.
+
+If you do not have CMake in a new enough version available, you may also install it manually as follows.
 
 .. code-block:: bash
 
   # you will need at least version 3.20.0 for GNU Compiler Collection 
   (cd $(mktemp -d) && wget -qO- https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-Linux-x86_64.tar.gz | tar -xvz -C "." && mv "./cmake-3.20.0-linux-x86_64" "${HOME}/bin/cmake")
```

Comment:
It's `${HOME}/bin/make` here, but `${PREFIX}/bin/...` below

Reply:
Thanks! Missed that one. Maybe `PREFIX` should be `SEISSOL_PREFIX` instead, that would make it more easy to recognize. The choice for `PREFIX` was mainly since if some `make` script has a prefix variable, it's sometimes named `PREFIX`, i.e. setting the environment variable in advance will spare us from setting that explicitly again in this case. ... But we actually don't really run into that, so... I'll rename it.